pom.xml

```diff
@@ -54,6 +54,18 @@
                         <artifactId>spring-boot-starter-tomcat</artifactId>
                         <scope>compile</scope>
               </dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.16.20</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.4</version>
+		</dependency>
+
```

Comment:
Has these new components been Foss checked/reported?

Reply:
How can i checked/reported? working with knouckout and need gson

Reply:
We have meeting on Wednesday about FOSSing new components. But if you want to do it directly you need to talk/chat with emichaf.